services/auth/src/api.js

```diff
@@ -1060,6 +1060,9 @@ builder.declare({
     req.clientId(),
     req.scopes(),
   ]);
+  //setting headers as purposed by bug 161375
+  req.params.test_auth = {clientId, scopes};
```

Comment:
I'm not sure why this is *setting* something in `req.params`.  The request should be read-only..